guide/additional-info/async-await.md

```diff
@@ -20,9 +20,9 @@ One important thing to know is that a Promise can only have one state at a time;
 
 ```js
 function deleteMessages(amount) {
-	return new Promise((resolve, reject) => {
-		if (amount > 10) return reject(new Error('You can\'t delete more than 10 Messages at a time.'));
-		resolve('Deleted 10 messages.');
+	return new Promise((resolve) => {
+		if (amount > 10) throw new Error('You can\'t delete more than 10 Messages at a time.');
+		setTimeout(() => resolve('Deleted 10 messages.'), 2000)
```

Comment:
this message is still incongruent with what the text below says it resolves with